selfdrive/car/subaru/carstate.py

```diff
@@ -13,6 +39,8 @@ def __init__(self, CP):
     can_define = CANDefine(DBC[CP.carFingerprint]["pt"])
     self.shifter_values = can_define.dv["Transmission"]["Gear"]
 
+    self.cruise_buttons = deque([Buttons.NONE] * PREV_BUTTON_SAMPLES, maxlen=PREV_BUTTON_SAMPLES)
```

Comment:
do you need a deque?

Reply:
was copying hyundai, but i think we only need 1 sample really https://github.com/commaai/openpilot/blob/2ebd7ab088ade61bbf661c140483fc477d444bc2/selfdrive/car/hyundai/carstate.py#L22